tests/Functional/Admin/BrowserTest.php

```diff
@@ -121,6 +121,7 @@ private function extractUniqId(Form $form): string
         \assert(\is_string($queryString));
 
         parse_str($queryString, $queryParameters);
+        \assert(\is_string($queryParameters['uniqid']));
```

Comment:
@VincentLanglet I just let this change, because phpstan was complain about it!

Reply:
Yes sure